src/views/Categories/components/EditForm/index.jsx

```diff
@@ -44,10 +58,12 @@ const EditForm = props => {
       >
         <FormSelect
           id="fire-hazard"
+          value={isFireHazardStr ? 'Yes' : 'No'}
```

Comment:
`isFireHazardStr` is already `'Yes'` or  `'No'`, so no need for the ternary operator here. (In fact, `'No'` is truthy since it's a non-empty string :) So this might not work as expected)
```suggestion
          value={isFireHazardStr}
```

Reply:
Leftover from when `isFireHazardStr` was still boolean - changed!